docs/examples/cookbook/button-component.md

```diff
@@ -65,7 +65,7 @@ local function Button(
 		TextColor3 = COLOUR_TEXT,
 
 		BackgroundColor3 = scope:Spring(
-			scope:Computed(function(use)
+			scope:Computed(function(use, scope)
```

Comment:
Scope shouldn't be provided as it is never used

Reply:
```suggestion
			scope:Computed(function(use)
```

Reply:
reverted